Apps/Sandcastle/gallery/Picking.html

```diff
@@ -144,6 +144,50 @@
     }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
 });
 
+Sandcastle.addToolbarButton('Pick position', function() {
```

Comment:
Yup, its picking the depth plane. The regular pick to see if the model is under the mouse cursor checks a 3x3 pixel region. Picking the position only uses the center pixel.